src/main/java/auto_ckz/common/enums/RepairStatusWrapper.java

```diff
@@ -0,0 +1,20 @@
+package auto_ckz.common.enums;
+
+
+import java.util.ArrayList;
+
+public class RepairStatusWrapper {
```

Comment:
Aaa jak już to wyjdzie pogmatwany kod w Thymeleaf'e, to może raczej bardziej czytelnie jest teraz